library/src/main/java/com/datatorrent/lib/appdata/query/serde/DataQuerySnapshotDeserializer.java

```
@@ -85,10 +123,10 @@ private Message deserializeHelper(String json,
 
```

Should this throw an exception instead of returning null?